src/config.rs

```diff
@@ -1,47 +1,48 @@
+use once_cell::sync::OnceCell;
 use serde::Deserialize;
 
+pub static CONFIG: OnceCell<Config> = OnceCell::new();
```

Comment:
Are you sure it's a good idea to have a global config as a once cell? I think it would be more flexible to pass `Config`/`&Config` everywhere :thinking:

Reply:
It might be, yeah, good point

Though the thing is we would have to plague the different function arguments/struct fields with Config/&Config
And that is what I wanted to avoid by using once cell.

Reply:
I also the same, we must not use global mutable vars if it not neccesary